### PR TITLE
1296 cleanup deprecation

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/casemodule/services/Services.java [hunk 1]
```diff
@@ -33,6 +33,7 @@
  * A collection of case-level services (e.g., file manager, tags manager,
  * keyword search, blackboard).
  */
+@SuppressWarnings("deprecation")
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/casemodule/services/Services.java]: Why do we need this?

File: Core/src/org/sleuthkit/autopsy/casemodule/services/Services.java [hunk 2]
```diff
@@ -93,12 +94,25 @@ public KeywordSearchService getKeywordSearchService() {
...
     }
+    
+     /**
+     * Gets the case blackboard service for the current case.
+     *
+     * @return The blackboard service for the current case.
+     */
+    public Blackboard getCaseBlackboard() {
+        return blackboard;
+    }
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/casemodule/services/Services.java]: This method is returning org.sleuthkit.autopsy.casemodule.services.Blackboard. It should return the TSK Blackboard.

File: Core/src/org/sleuthkit/autopsy/modules/dataSourceIntegrity/DataSourceIntegrityIngestModule.java [hunk 3]
```diff
@@ -297,8 +299,11 @@ public ProcessResult process(Content dataSource, DataSourceIngestModuleProgress
                     BlackboardArtifact verificationFailedArtifact = Case.getCurrentCase().getSleuthkitCase().newBlackboardArtifact(BlackboardArtifact.ARTIFACT_TYPE.TSK_VERIFICATION_FAILED, img.getId());
                     verificationFailedArtifact.addAttribute(new BlackboardAttribute(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_COMMENT,
                         DataSourceIntegrityModuleFactory.getModuleName(), artifactComment));
-                    IngestServices.getInstance().fireModuleDataEvent(new ModuleDataEvent(DataSourceIntegrityModuleFactory.getModuleName(),
-                        BlackboardArtifact.ARTIFACT_TYPE.TSK_VERIFICATION_FAILED));
+                    try {
+                        Case.getCurrentCase().getSleuthkitCase().getBlackboard().postArtifact(verificationFailedArtifact, DataSourceIntegrityModuleFactory.getModuleName());
+                    } catch (Blackboard.BlackboardException ex) {
+                        logger.log(Level.WARNING, String.format("Unable to post artifact to Blackboard. artifactid: %d", verificationFailedArtifact.getArtifactID()), ex);
+                    }
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/modules/dataSourceIntegrity/DataSourceIntegrityIngestModule.java]: Modules should go through Services.

File: Core/src/org/sleuthkit/autopsy/test/CustomArtifactType.java [hunk 4]
```diff
@@ -65,7 +65,7 @@
      * @throws BlackboardException If there is an error adding any of the types.
      */
     static void addToCaseDatabase() throws Blackboard.BlackboardException, NoCurrentCaseException {
-        Blackboard blackboard = Case.getCurrentCaseThrows().getServices().getBlackboard();
+        Blackboard blackboard = Case.getCurrentCaseThrows().getSleuthkitCase().getBlackboard();
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/test/CustomArtifactType.java]: Modules should go through Services.

File: Core/src/org/sleuthkit/autopsy/test/InterestingArtifactCreatorIngestModule.java [hunk 5]
```diff
@@ -55,7 +55,7 @@
     @Override
     public void startUp(IngestJobContext context) throws IngestModuleException {
         try {
-            Blackboard blackboard = Case.getCurrentCaseThrows().getServices().getBlackboard();
+            Blackboard blackboard = Case.getCurrentCaseThrows().getSleuthkitCase().getBlackboard();
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/test/InterestingArtifactCreatorIngestModule.java]: Modules should go through Services.

File: Core/src/org/sleuthkit/autopsy/test/InterestingArtifactCreatorIngestModule.java [hunk 6]
```diff
@@ -77,7 +77,7 @@ public ProcessResult process(AbstractFile file) {
              * type.
              */
             int randomArtIndex = (int) (Math.random() * 3);
-            Blackboard blackboard = Case.getCurrentCaseThrows().getServices().getBlackboard();
+            Blackboard blackboard = Case.getCurrentCaseThrows().getSleuthkitCase().getBlackboard();
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/test/InterestingArtifactCreatorIngestModule.java]: Modules should go through Services.

File: Core/src/org/sleuthkit/autopsy/timeline/ui/listvew/datamodel/ListViewModel.java [hunk 7]
```diff
@@ -82,12 +82,12 @@ public ListViewModel(FilteredEventsModel eventsModel) {
     public List<CombinedEvent> getCombinedEvents(Interval timeRange, RootFilterState filterState) throws TskCoreException {
         List<TimelineEvent> events = eventManager.getEvents(timeRange, filterState.getActiveFilter());
         
+        ArrayList<CombinedEvent> combinedEvents = new ArrayList<>();
+        
         if (events == null || events.isEmpty()) {
-            return Collections.EMPTY_LIST;
+            return combinedEvents; //return an empty list;
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/timeline/ui/listvew/datamodel/ListViewModel.java]: Please elaborate on the reason for making these changes.

**Comment by rcordovano** [comment]
Please resolve Codacy issue.

File: Core/src/org/sleuthkit/autopsy/timeline/ui/listvew/datamodel/ListViewModel.java (hunk 7 above)
Comment (kellykelly3) [reply Core/src/org/sleuthkit/autopsy/timeline/ui/listvew/datamodel/ListViewModel.java]: There was a warning about returning an untyped list.

**Pull request closed without merging**